### DISCUSSION TITLE
Editieren - Koordinateneingabe eines Vertex über "Koordinaten (absolut)" führt zu Fehlermeldung

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#134] [Q&A]
Beim **Anlegen** einer Fläche im Editiermodus ist es möglich über Koordinaten (absolut) Vertices einzutragen. 
Das funktioniert einwandfrei.
Wenn man einen Vertex dieser Fläche über Koordinaten (absolut) verändert und speichern will, bekommt man die Fehlermeldung:
"Input string was not in a correct format."
Verändert man diesen Vertex manuell (verschieben des Vertex) ist es wieder problemlos möglich die Fläche zu speichern.
In den DevTools -> Nutzlast -> eventString erkennt man, dass sich der String bei Koordinativer Eingabe zur manuellen Änderung unterscheidet. (Beim Zeichen 2417 fangen die Unterschiede an).

**Änderung des Vertex über Koordinaten (absolut)**
edit-map-scale=935|edit-map-crsid=31254|_editfield_themeid=9a1367e1b1ce48d5873b2ebe371b9842|_editfield_edittheme_def=digi_fwp_gv,19,9a1367e1b1ce48d5873b2ebe371b9842,Fläche|_editfield_oid=5118|_editfield_globaloid=digi_fwp_gv:aktenzahl:5118|editfield_email_bearbeiter=juergen.oberressl@vorarlberg.at|editfield_erstellt_am=13.12.2023|editfield_gemeinde=Bartholomäberg|editfield_gem_nr=80101|editfield_wi_em_kn=BB-I|editfield_wi_em_kn_idx=|editfield_wi_em_ukn=|editfield_f_kn=|editfield_fw_kn=|editfield_f_ukn=|editfield_fs_txt=|editfield_wi_em_sig=|editfield_mass_bauliche_nutzung=|editfield_stand=15.12.2023|editfield_bearbeiter=juergen.oberressl@vorarlberg.at|editfield_flaeche=12289,87|editfield_gst_nr=90101-1286/1;90101-1283/1|editfield_gem_akt=Bartholomäberg_47114756123_rek|editfield_vkz=80000|editfield_abteilung=viia|editfield_aktenzahl=47114756123_rek|edit-theme-select=#|edit-theme-insert=digi_fwp_gv,19,9a1367e1b1ce48d5873b2ebe371b9842|coordinates-input-x-value=-34025,7|coordinates-input-y-value=217614,6|_method=save|_sketch=MULTIPOLYGON((-33995.649992751605+217512.23000039905,-34006.74999954406+217518.79999696836,-34018.31999743526+217522.41000313498,-34019.565999656304+217518.41099844407,-34022.83699405071+217507.91200241726,-34023.51999784838+217505.72000349965,-34024.31199900548+217503.98599747568,-34025.87999208638+217500.5499973325,-34026.89799926217+217499.99999512266,-34031.02499297827+217497.76899847202,-34031.1900004079+217497.68000016175,-34033.62599538325+217499.18299996667,-34034.948998638705+217500.0000016177,-34035.49999484348+217500.34000213724,-34042.989992550254+217506.25000450853,-34050.65999743525+217512.1199976094,-34057.1199914444+217516.39000098966,-34066.44000041911+217522.3700017212,-34074.99000053666+217527.72999720648,-34081.65999870236+217531.46000423003,-34082.87999413312+217532.15000255778,-34092.67999650865+217538.00000121817,-34097.81999525273+217541.51000086218,-34102.949991395755+217545.61000062246,-34106.59999218568+217549.41000072472,-34109.97000078509+217553.44000166934,-34113.53999558218+217558.07999715488,-34115.19999527425+217560.90000425652,-34120.12999778804+217569.10999799613,-34126.800000571246+217579.709995199,-34121.67699481112+217584.03399982583,-34113.52999828562+217590.91999582388,-34105.8639964732+217617.0409984067,-34104.95099931495+217620.14999535773,-34102.881000442896+217618.90999908932,-34076.39999333185+217603.03999883402,undefined+undefined,-33938.18999521854+217527.87000157684,-33900.549993694294+217512.46000308823,-33910.969992750666+217502.32000205014,-33913.51999349186+217499.999999593,-33934.809996947384+217480.01000500843,-33935.36000098341+217479.47000332642,-33958.06999824775+217488.42999863997,-33948.77999242123+217500.00000479817,-33943.32999873972+217506.80000384524,-33972.409994025504+217520.70999525953,-33973.43999649479+217516.386003525,-33976.63999938396+217502.95999585185,-33986.48999814445+217506.4299955452,-33985.67999925505+217508.72000273038))|_sketchWgs84=MULTIPOLYGON((9.885249928617421+47.09566764780827,9.885103228485454+47.09572616720814,9.884950562552477+47.09575803783886,9.884934454970358+47.09572200338138,9.884892169649985+47.095627398436775,9.884883340213667+47.09560764664326,9.884873040347884+47.095592008695945,9.884852649062841+47.09556102156647,9.884839282451923+47.09555602163238,9.884785094432+47.09553574030171,9.88478292784623+47.095534931224414,9.884750728548207+47.09554832378818,9.884733240858322+47.095555603783254,9.884725957711824+47.09555863337769,9.884626855821217+47.09561140312726,9.884525385856612+47.09566380356989,9.884439974785323+47.095701875370914,9.884316763427059+47.09575517939531,9.884203741024947+47.09580294655371,9.884115604581137+47.09583615009682,9.88409948312648+47.09584229299379,9.883969958860932+47.095894402360074,9.883901990995044+47.09592570640054,9.883834109856082+47.09596231778697,9.883785745001507+47.095996307724995,9.883741050437+47.09603238099526,9.883693675243528+47.096073930551505,9.883671595909187+47.096099209254,9.883606035359046+47.09617279901731,9.883517374206324+47.09626779550112,9.883584521848913+47.09630695521944,9.883691304647163+47.09636931656463,9.883790286972738+47.09660466563557,9.883802075667234+47.09663267758312,9.883829435107314+47.09662163188744,9.884179437507006+47.09648026317885,9.884846351920958+47.09658687511803,9.886005563766306+47.09581130182076,9.886502498366982+47.095674641107586,9.886366021836935+47.095582896030805,9.886332610896865+47.09556189642007,9.886053710377219+47.09538099093431,9.886046507098746+47.095376105316845,9.885746709788554+47.09545552178378,9.88586819308695+47.09556007153536,9.885939460695676+47.0956215176159,9.88555538526948+47.09574512741518,9.885542146935215+47.09570618107704,9.885501017726263+47.095585252470514,9.885371017596325+47.09561595348721,9.885381512459452+47.095636593380995))|_sketchSrs=31254|_sketchInfo={"geometryType":"polygon"}|mapcrs=31254|_mapcrs=31254|_calccrs=31254|_deviceinfo={"is_mobile_device":false,"screen_width":1920,"screen_height":1200,"advanced_tool_behaviour":true}

**Änderung über manuelles Verschieben:**
edit-map-scale=935|edit-map-crsid=31254|_editfield_themeid=9a1367e1b1ce48d5873b2ebe371b9842|_editfield_edittheme_def=digi_fwp_gv,19,9a1367e1b1ce48d5873b2ebe371b9842,Fläche|_editfield_oid=5118|_editfield_globaloid=digi_fwp_gv:aktenzahl:5118|editfield_email_bearbeiter=juergen.oberressl@vorarlberg.at|editfield_erstellt_am=13.12.2023|editfield_gemeinde=Bartholomäberg|editfield_gem_nr=80101|editfield_wi_em_kn=BB-I|editfield_wi_em_kn_idx=|editfield_wi_em_ukn=|editfield_f_kn=|editfield_fw_kn=|editfield_f_ukn=|editfield_fs_txt=|editfield_wi_em_sig=|editfield_mass_bauliche_nutzung=|editfield_stand=15.12.2023|editfield_bearbeiter=juergen.oberressl@vorarlberg.at|editfield_flaeche=12289,87|editfield_gst_nr=90101-1286/1;90101-1283/1|editfield_gem_akt=Bartholomäberg_47114756123_rek|editfield_vkz=80000|editfield_abteilung=viia|editfield_aktenzahl=47114756123_rek|edit-theme-select=#|edit-theme-insert=digi_fwp_gv,19,9a1367e1b1ce48d5873b2ebe371b9842|coordinates-input-x-value=-34025,7|coordinates-input-y-value=217614,6|_method=save|_sketch=MULTIPOLYGON((-33995.649992751605+217512.23000039905,-34006.74999954406+217518.79999696836,-34018.31999743526+217522.41000313498,-34019.565999656304+217518.41099844407,-34022.83699405071+217507.91200241726,-34023.51999784838+217505.72000349965,-34024.31199900548+217503.98599747568,-34025.87999208638+217500.5499973325,-34026.89799926217+217499.99999512266,-34031.02499297827+217497.76899847202,-34031.1900004079+217497.68000016175,-34033.62599538325+217499.18299996667,-34034.948998638705+217500.0000016177,-34035.49999484348+217500.34000213724,-34042.989992550254+217506.25000450853,-34050.65999743525+217512.1199976094,-34057.1199914444+217516.39000098966,-34066.44000041911+217522.3700017212,-34074.99000053666+217527.72999720648,-34081.65999870236+217531.46000423003,-34082.87999413312+217532.15000255778,-34092.67999650865+217538.00000121817,-34097.81999525273+217541.51000086218,-34102.949991395755+217545.61000062246,-34106.59999218568+217549.41000072472,-34109.97000078509+217553.44000166934,-34113.53999558218+217558.07999715488,-34115.19999527425+217560.90000425652,-34120.12999778804+217569.10999799613,-34126.800000571246+217579.709995199,-34121.67699481112+217584.03399982583,-34113.52999828562+217590.91999582388,-34105.8639964732+217617.0409984067,-34104.95099931495+217620.14999535773,-34102.881000442896+217618.90999908932,-34076.39999333185+217603.03999883402,-34053.74948222552+217630.9792400198+meta:changed,-33938.18999521854+217527.87000157684,-33900.549993694294+217512.46000308823,-33910.969992750666+217502.32000205014,-33913.51999349186+217499.999999593,-33934.809996947384+217480.01000500843,-33935.36000098341+217479.47000332642,-33958.06999824775+217488.42999863997,-33948.77999242123+217500.00000479817,-33943.32999873972+217506.80000384524,-33972.409994025504+217520.70999525953,-33973.43999649479+217516.386003525,-33976.63999938396+217502.95999585185,-33986.48999814445+217506.4299955452,-33985.67999925505+217508.72000273038))|_sketchWgs84=MULTIPOLYGON((9.885249928617421+47.09566764780827,9.885103228485454+47.09572616720814,9.884950562552477+47.09575803783886,9.884934454970358+47.09572200338138,9.884892169649985+47.095627398436775,9.884883340213667+47.09560764664326,9.884873040347884+47.095592008695945,9.884852649062841+47.09556102156647,9.884839282451923+47.09555602163238,9.884785094432+47.09553574030171,9.88478292784623+47.095534931224414,9.884750728548207+47.09554832378818,9.884733240858322+47.095555603783254,9.884725957711824+47.09555863337769,9.884626855821217+47.09561140312726,9.884525385856612+47.09566380356989,9.884439974785323+47.095701875370914,9.884316763427059+47.09575517939531,9.884203741024947+47.09580294655371,9.884115604581137+47.09583615009682,9.88409948312648+47.09584229299379,9.883969958860932+47.095894402360074,9.883901990995044+47.09592570640054,9.883834109856082+47.09596231778697,9.883785745001507+47.095996307724995,9.883741050437+47.09603238099526,9.883693675243528+47.096073930551505,9.883671595909187+47.096099209254,9.883606035359046+47.09617279901731,9.883517374206324+47.09626779550112,9.883584521848913+47.09630695521944,9.883691304647163+47.09636931656463,9.883790286972738+47.09660466563557,9.883802075667234+47.09663267758312,9.883829435107314+47.09662163188744,9.884179437507006+47.09648026317885,9.88447566012766+47.0967327362773,9.886005563766306+47.09581130182076,9.886502498366982+47.095674641107586,9.886366021836935+47.095582896030805,9.886332610896865+47.09556189642007,9.886053710377219+47.09538099093431,9.886046507098746+47.095376105316845,9.885746709788554+47.09545552178378,9.88586819308695+47.09556007153536,9.885939460695676+47.0956215176159,9.88555538526948+47.09574512741518,9.885542146935215+47.09570618107704,9.885501017726263+47.095585252470514,9.885371017596325+47.09561595348721,9.885381512459452+47.095636593380995))|_sketchSrs=31254|_sketchInfo={"geometryType":"polygon"}|mapcrs=31254|_mapcrs=31254|_calccrs=31254|_deviceinfo={"is_mobile_device":false,"screen_width":1920,"screen_height":1200,"advanced_tool_behaviour":true}

Auf der Datenbank sind keine Fehlermeldungen erkennbar.
Beim Zeichnen (Redlining) tritt dieses Problem nicht auf.

**Comment by gis-eni** [comment]
Ja, irgendwo in deinen Koordinaten steht "undefined":

<img width="230" alt="image" src="https://github.com/gis-eni/webgis-forum/assets/62870568/1276e840-7fc9-4c9d-8d19-3aaa5347fad2">

Das Problem schein aufzutreten und kommt nur, wenn man ein bestehenden Vertex setzt. Ein neuer Vertex funktioniert... Schaut nach einem klassischen Bug aus. Werde noch weiter testen, ist aber mit dem nächsten Build behoben.
